Vagrantfile

```diff
@@ -11,7 +11,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     c.vm.box = "trusty64"
     c.vm.box_url = "https://cloud-images.ubuntu.com/vagrant/trusty/current/trusty-server-cloudimg-i386-vagrant-disk1.box"
 
-    c.vm.provision :shell, inline: "apt-get update"
+    c.vm.provision :shell, :path => "bootstrap.sh"
```

Comment:
I'd prefer a separate bootstrap script, since you have all the necessary commands at one place.